src/components/calcite-radio-button/calcite-radio-button.tsx

```diff
@@ -270,7 +269,7 @@ export class CalciteRadioButton {
   //--------------------------------------------------------------------------
 
   connectedCallback(): void {
-    this.documentRoot = this.el.getRootNode();
+    this.rootNode = this.el.getRootNode();
```

Comment:
You can eliminate some casting above by typing `rootNode` as `HTMLElement` and casting it to that type here:

```tsx
this.rootNode = this.el.getRootNode() as HTMLElement;
```